docs/Contributing/API-for-contributors.md

```diff
@@ -725,6 +727,128 @@ This replaces the active global enroll secrets with the secrets specified.
 
 `Status: 200`
 
+### Search targets
```

Comment:
How do we decide which endpoints are documented in the REST API docs, and which are documented in the API for contributors?